server/routerlicious/packages/services-client/src/test/types/validateServerServicesClientPrevious.generated.ts

```diff
@@ -41,7 +41,7 @@ use_current_ClassDeclaration_BasicRestWrapper(
 declare function get_current_ClassDeclaration_BasicRestWrapper():
     TypeOnly<current.BasicRestWrapper>;
 declare function use_old_ClassDeclaration_BasicRestWrapper(
-    use: TypeOnly<old.BasicRestWrapper>);
+    use: TypeOnly<old.BasicRestWrapper>): void;
```

Comment:
I'm curious what caused these files to change. Explicit return types seem like an improvement, just not sure what triggered the change.